spec/acts_as_taggable_on/tag_spec.rb

```diff
@@ -2,8 +2,11 @@
 require 'spec_helper'
 require 'db/migrate/2_add_missing_unique_indices.rb'
 
+
 shared_examples_for 'without unique index' do
-  prepend_before(:all) { AddMissingUniqueIndices.down }
+  prepend_before(:all) do
```

Comment:
When I implemented the patch as a migration I added the new one there, but then, after changing perspective (as the migration was an "execute", not tracked in the schema) I removed both migration and its call in the tag_spec.rb. Thus that changed. But I can reset the previous line.